### PR TITLE
[Windows Support] [IntelTensorFlow_GettingStarted] enable windows support on the sample

### PR DESCRIPTION
**Pull request by yinghu5**
[Windows Support] [IntelTensorFlow_GettingStarted] enable windows support on the sample
1.  Add window CI test
2. Modify oneDNN verbose with latest v2.5

### CODE REVIEW

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "This sample illustrates how to train a TensorFlow model and run inference with oneMKL and oneDNN.",
   "builder": ["cli"],
   "languages": [{"python":{}}],
-  "os":["linux"],
+  "os":["linux",, "windows"],
```
Comment (jimmytwei) [inline AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json:8]: I see two commas here. Can you please validate sample.json?

Comment (yinghu5) [reply AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json]: I have updated the file. Please help to review.  thanks

**Review by ishaghosh27 — APPROVED** [review]

**Comment by yinghu5** [comment]
Windows output: [Log from tensorflow-2.8 by pip]
WARNING:tensorflow:From C:\Users\xxx\Miniconda3\envs\tensorflow-pip\lib\site-packages\tensorflow\python\compat\v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term

2022-04-13 17:18:33.573793: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
0 0.39215377
1 0.35312158
2 0.34055027
3 0.33485362
4 0.33175677
Tensorflow HelloWorld Done!
[CODE_SAMPLE_COMPLETED_SUCCESFULLY]

and the DNNL_VERBOSE=1

>set DNNL_VERBOSE=1
>python TensorFlow_HelloWorld.py
WARNING:tensorflow:From C:\Users\huangya1\Miniconda3\envs\tensorflow-pip\lib\site-packages\tensorflow\python\compat\v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
2022-04-14 17:04:55.455806: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
onednn_verbose,info,oneDNN v2.5.0 (commit N/A)
onednn_verbose,info,cpu,runtime:OpenMP
onednn_verbose,info,cpu,isa:Intel AVX-512 with Intel DL Boost
onednn_verbose,info,gpu,runtime:none

File: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json [hunk 2]
```diff
@@ -17,5 +17,14 @@
 		 ]
 	}
      ]
```
Comment (tingleby) [inline AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json:19]: This file is not valid JSON.

Comment (jimmytwei) [reply AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json:19]: Thanks for the catch. I've reverted the PR and will let the sample owner know about the issue.

**Comment by yinghu5** [comment]
fix the json issue.  add one comma , thanks

**Pull request merged**